pkg/scheduler/core/extender.go

```diff
@@ -287,7 +289,7 @@ func (h *HTTPExtender) Filter(
 	}
 
 	if h.filterVerb == "" {
-		return nodes, extenderv1.FailedNodesMap{}, nil
+		return nodes, nil, nil, nil
```

Comment:
I'm wondering this would break when loop the failedNodesMap in "genernicSchduler.findNodesThatPassExtenders"

Reply:
Thanks! Fixed.